### PR TITLE
feat: add sandbox generation PE-1440

### PR DESCRIPTION
**Pull request by jdaev**

### CODE REVIEW

**Comment by fedellen** [comment]
There appears to be an empty file committed: `sanbox_test.dart`

File: lib/src/api/sandbox.dart [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      if (bc % 4 > 0) {
+        bs = bs * 64 + buffer;
+        bc++;
+        output += String.fromCharCode(255 & (bs >> ((-2 * bc) & 6)));
+      } else {
+        bs = buffer;
+        bc++;
+      }
+    }
+  }
```
Comment (fedellen) [inline lib/src/api/sandbox.dart:65]: With the test case wrapped around this, we have the opportunity to improve the readability here

**Review by fedellen — DISMISSED** [review]
LGTM, thanks @jdaev !

**Review by matibat — DISMISSED** [review]
The tests includes examples for numbers, capital and lowercase letters, dash and underscores for both the input and output. Looking good!

**Review by thiagocarvalhodev — DISMISSED** [review]

**Review by agsuy — APPROVED** [review]
Now points to dev branch.

**Pull request merged**